.github/workflows/kitchen.yml

```diff
@@ -224,3 +223,35 @@ jobs:
           cd /home/runner/work/chef/chef/kitchen-tests
           bundle install
           bundle exec kitchen test end-to-end-${{ matrix.os }}
+  linux-2004-host:
```

Comment:
Can we add a comment here stating we have to use 20.04 host for these operating other wise the dokken images throw timedatectl error, so that in future we would have quick reference when making any changes here

Reply:
Done.